sdk/storage/Azure.Storage.Blobs.Batch/src/Azure.Storage.Blobs.Batch.csproj

```diff
@@ -4,6 +4,8 @@
   </PropertyGroup>
   <PropertyGroup>
     <AssemblyTitle>Microsoft Azure.Storage.Blobs.Batch client library</AssemblyTitle>
+    <AssemblyTitle>Microsoft Azure.Storage.Blobs.Batching client library</AssemblyTitle>
```

Comment:
Looks like a bad merge here where the title was duplicated.

Reply:
Thanks. Fix is here pending approval: https://github.com/Azure/azure-sdk-for-net/pull/8496